tornado/test/options_test_types.cfg

```diff
@@ -0,0 +1,16 @@
+from datetime import datetime, timedelta
+import sys
+import os
+sys.path.insert(0,
```

Comment:
Is this necessary? I don't think it should be; the tornado directory should already be on the path. Individual tests should avoid side effects like this, so if we can't test this without manipulating the path I'd prefer to just skip the test of a custom class.